rdkit/Chem/Draw/IPythonConsoleIntegration.py

```diff
@@ -0,0 +1,424 @@
+
+"""RDKit Conformer Browser
+Derived from Greg Landrum's code by Malitha Humayun Kabir
+As a part of GSoC 2017
+Project : RDKit - 3Dmol.js integration
+Mentors: Paul Czodrowski and Greg Landrum
+Date: 17th July 2017
+Email# malitha12345@gmail.com
+"""
+
+import py3Dmol
+from rdkit import Chem
+from rdkit.Chem import AllChem
+from rdkit.Chem import Descriptors
+from ipywidgets import Layout, Label, Button, Box, HBox, VBox
+from ipywidgets import Dropdown, SelectMultiple, IntSlider, HTML, Checkbox, Button
+from IPython.display import display
+import time
+
+
+bgcolors_3d = ['0xeeeeee', '0x000000', '0xffffff']
+
+prop_rdkit=sorted([prop for prop,obj in Descriptors._descList])
+
+drawing_types_3d=['line', 'cross', 'stick', 'cartoon', 'sphere', 'surface', 'ballstick']
+
+color_scheme_3d=['default', 'greenCarbon', 'cyanCarbon', 'magentaCarbon', 
+                 'yellowCarbon', 'whiteCarbon', 'orangeCarbon', 'purpleCarbon', 
+                 'blueCarbon', 'ssPyMOL', 'ssJmol', 'Jmol', 'amino', 
+                 'shapely', 'nucleic', 'chain', 'chainHetatm', 'prop']
+
+
+def ProcessMolContainingObj(Mol):
+    """This function checks whether the object type fits the requirements for rendering.
+    If the oject doesn't have necessary attributes, it takes action to include that.
+    """
+    try:
+        moldict = dict(Mol)
+    except TypeError:
+        if type(Mol) is tuple:
+            moldict=list()
+            moldict.append(Mol)
+            moldict = dict(moldict)
```

Comment:
This suggestion also followed in most recent commit. 
Yes, `moldict = {mol[0]: mol[1]}` is the correct and readable writing style.